app/main.py

```diff
@@ -1,3 +1,27 @@
+'''
+MIT License
```

Comment:
do we need this on all files? feels overkill

Reply:
I am not sure tbh, if some source code is taken from a repo using a license like MIT it is recommended that we add it to the source file if the binaries are not being distributed. Also, it is just two files